src/gui/editors/AutomationEditor.cpp

```diff
@@ -398,6 +398,58 @@ void AutomationEditor::drawLine( int x0In, float y0, int x1In, float y1 )
 
 
 
+bool AutomationEditor::fineTuneValue(timeMap::iterator node, bool editingInValue)
+{
+	if (node != m_pattern->getTimeMap().end())
+	{
+		// Display dialog to edit the value
+		bool ok;
+		double value = QInputDialog::getDouble(
+			this,
+			tr("Edit Value"),
+			editingInValue
+				? tr("New inValue")
+				: tr("New outValue"),
+			editingInValue
+				? INVAL(node)
+				: OUTVAL(node),
+			m_pattern->firstObject()->minValue<float>(),
+			m_pattern->firstObject()->maxValue<float>(),
+			3,
+			&ok
+		);
+
+		if (ok)
+		{
+			// Set the new inValue/outValue
+			if (editingInValue)
+			{
+				// If the outValue is equal to the inValue we
+				// set both to the given value
+				if (OFFSET(node) == 0)
+				{
+					node.value().setOutValue(value);
+				}
+				node.value().setInValue(value);
+				Engine::getSong()->setModified();
```

Comment:
We `setModified` in both branches of the `if`, we can move this line to after the if/else.